### ISSUE TITLE
Stopped working in IE 11

### ISSUE THREAD

**Issue by sebastianhelbig** [mirego/ember-best-language#17]
Version 0.1.0 worked fined. With recent version 1.0.3 I get the following:

> TypeError: Invalid attempt to spread non-iterable instance

This stops the application from running completely. This only happens in IE11, it works fine in Chrome.

Code causing this is in beforeModel of application route:

```
config.i18n.allowedLocales = ['de', 'en', 'fr'];
const bestLanguage = get(this, 'bestLanguage').bestLanguageOrFirst(config.i18n.allowedLocales);
const languageToSet = get(bestLanguage, 'baseLanguage');
```

**Comment by scottkidder** [comment]
What do you have in your `config/targets.js`?

**Comment by sebastianhelbig** [comment]
```
'use strict';

const browsers = [
  '> 1%',
  'last 2 versions',
  'Android >= 5',
  'IE >= 11',
  'iOS >= 9'
];

module.exports = {
  browsers
};
```

**Comment by charlesdemers** [comment]
This was fixed by #18 🎉 Please install version 1.0.4 to get the fix
